src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+@import '/src/styles/blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
+}
+
+body {
+  margin: 0;
+}
+
+.cards {
+  padding: 50px 40px;
+  display: grid;
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
not fixed 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/03ae777f-f8da-4d82-a6ed-c990e6d490ab)


```suggestion

* {
 box-sizing: border-box;
}

.card {
```